internal/codegen/tpl.go

```diff
@@ -23,7 +23,7 @@ func NewFuncMap(st *Stencil, t *Template, log logrus.FieldLogger) template.FuncM
 	if st != nil {
 		tplst = &TplStencil{st, t, log}
 	}
-	if t != nil {
+	if t != nil && len(t.Files) > 0 {
```

Comment:
Hmmm, what was causing this case? By default a template _always_ creates a file (at least at this call time, it does). This is due to the default file logic. So, I'm a little confused where `t.Files` is empty.

If if this does make it safe, now `file` calls would panic, which I don't think would be expected either.

Reply:
This was actually because of the change in the other PR, where it's rendering basically one line of a template, but not producing a file.

Reply:
Chatted more offline, not _super_ sure why this is working but w/e.